hydra-node/test/Hydra/Chain/Direct/StateSpec.hs

```diff
@@ -389,15 +389,16 @@ forAllFanout action =
        in action utxo tx
             & label ("Fanout size: " <> prettyLength (countAssets $ txOuts' tx))
  where
-  maxSupported = 67
+  maxSupported = 42
```

Comment:
According to the benchmarks, `52` should be possible (ball-park, not reproducible). Please check again what a maximum supported value for this would be (for `maximumNumberOfParties`)

Reply:
I've set that to 52 and run `cabal bench tx-cost` and observed 50 instead of 46 in _Cost of FanOut Transaction_ array.

Don't know if it matches what you expected.